src/GO/Job.php

```diff
@@ -2,6 +2,7 @@
 
 use DateTime;
 use Exception;
+use Cron\CronExpression;
```

Comment:
This is the line I'd like you to move to the `@var CronExpression` down in the code.